src/Lykke.Service.BlockchainCashoutPreconditionsCheck/Controllers/BlackListController.cs

```diff
@@ -0,0 +1,181 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Lykke.Common.Api.Contract.Responses;
+using Lykke.Common.ApiLibrary.Contract;
+using Lykke.Service.BlockchainCashoutPreconditionsCheck.Core.Domain.Validation;
+using Lykke.Service.BlockchainCashoutPreconditionsCheck.Core.Exceptions;
+using Lykke.Service.BlockchainCashoutPreconditionsCheck.Core.Services;
+using Lykke.Service.BlockchainCashoutPreconditionsCheck.Filter;
+using Lykke.Service.BlockchainCashoutPreconditionsCheck.Models.Requests;
+using Lykke.Service.BlockchainCashoutPreconditionsCheck.Models.Responses;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace Lykke.Service.BlockchainCashoutPreconditionsCheck.Controllers
+{
+    [Route("api/[controller]")]
+    public class BlackListController : Controller
+    {
+        private readonly char[] _charactersToTrim = new char[] {' ', '\t'};
+        private readonly IBlackListService _blackListService;
+
+        public BlackListController(IBlackListService blackListService)
+        {
+            _blackListService = blackListService;
+        }
+
+        /// <summary>
+        /// is address black listed
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet("is-blocked/{blockchainType}/{blockedAddress}")]
```

Comment:
{blockchainType}/{blockedAddress}/is-blocked would be more RESTful

Reply:
fixed